taskt/Core/Automation/Commands/Word Commands/WordAddDocumentCommand.cs

```diff
@@ -24,7 +24,7 @@ public class WordAddDocumentCommand : ScriptCommand
 
         public WordAddDocumentCommand()
         {
-            this.CommandName = "WordAddDocumentCommand";
+            this.CommandName = "AddDocumentCommand";
```

Comment:
I noticed you've removed "Excel" from the Excel command file names but you're keeping "Word" in the Word commands. We should make this consistent. Also, in order to avoid potential naming conflicts with the DataTable Commands, we should probably keep "Excel" in the Excel command file names.

Reply:
done